### DISCUSSION TITLE
Howto backup and restore etcd with talos

### DISCUSSION THREAD

**Discussion by udoL** [siderolabs/talos#3439] [Q&A] [answered]
Hi,
I tried talos and it's looks nice. But I'm not sure how to backup etcd to be able to recover an talos-cluster in case of emergency.
I can copy snap and db-files from the etcd-members, but how works an restore?
talosctl list -l /var/lib/etcd/member/snap
NODE           MODE         SIZE(B)   LASTMOD           NAME
10.10.10.11   drwx------   246       Apr  6 04:50:06   .
10.10.10.11   -rw-r--r--   18228     Apr  4 08:40:59   0000000000000011-000000000011af78.snap
10.10.10.11   -rw-r--r--   18228     Apr  4 19:43:15   0000000000000011-0000000000133619.snap
10.10.10.11   -rw-r--r--   18227     Apr  5 06:45:27   0000000000000011-000000000014bcba.snap
10.10.10.11   -rw-r--r--   18228     Apr  5 17:47:50   0000000000000011-000000000016435b.snap
10.10.10.11   -rw-r--r--   18228     Apr  6 04:50:04   0000000000000011-000000000017c9fc.snap
10.10.10.11   -rw-------   5742592   Apr  6 10:34:19   db

talosctl etcd is for managing members only.
Is there an talos-way or must it be done with etcdctl from outside?

Regards
Udo

**Comment by smira** [comment]
Hi @udOl!

The backup and restore for etcd is coming in Talos 0.10 release, scheduled by the end of this month

**Reply by udoL** [reply]
Hi Andrey,
that sounds good!

Best regards
Udo

**Comment by steverfrancis** [comment, marked as answer]
Hi @udoL 
Hopefully you noticed that etcd backup/recovery was released in v0.10 - if not, here are the docs.
https://www.talos.dev/docs/v0.11/guides/disaster-recovery/
Let us know if you have any issues.
